smk-config.json

```diff
@@ -101,7 +101,7 @@
                 "fillColor": "#d95f02",
                 "fillOpacity": 0.5
             },
-            "popupTemplate": "<div class=\"smk-body\">  <div class=\"smk-feature\"><!----> <div class=\"smk-layer\">Municipal Regional District Tax: 2%</div> <div class=\"smk-attributes\"><div class=\"smk-attribute\"><span class=\"smk-attribute-title\">Recipient</span> <span><span class=\"smk-value\">{{feature.properties.Recipient}}</span></span></div><div class=\"smk-attribute\"><span class=\"smk-attribute-title\">Tax Rate</span> <span><span class=\"smk-value\">{{feature.properties.Tax_Rate}}</span></span></div><div class=\"smk-attribute\"><span class=\"smk-attribute-title\">Accom<wbr>_Area</span> <span><span class=\"smk-value\">{{feature.properties.Accom_Area}}</span></span></div><div class=\"smk-attribute\"><span class=\"smk-attribute-title\">Effect<wbr>_date</span> <span><span class=\"smk-value\">{{(new Date(feature.properties.Effect_date)).toLocaleDateString()}}</span></span></div><!----></div></div></div>"
+            "popupTemplate": "<div class=\"smk-body\">\n    <div class=\"smk-feature\">\n      <!---->\n        <div class=\"smk-layer\">\n            Municipal Regional District Tax: 2%\n        </div>\n        <div class=\"smk-attributes\">\n            <div class=\"smk-attribute\">\n                <span class=\"smk-attribute-title\">\n                    Recipient:\n                </span>\n                <span>\n                    <span class=\"smk-value\">\n                        {{feature.properties.Recipient}}\n                    </span>\n                </span>\n            </div>\n            <div class=\"smk-attribute\">\n                <span class=\"smk-attribute-title\">\n                    Tax Rate:\n                </span>\n                <span>\n                    <span class=\"smk-value\">\n                        {{feature.properties.Tax_Rate}}\n                    </span>\n                </span>\n            </div>\n            <div class=\"smk-attribute\">\n                <div class=\"smk-attribute\">\n                    <span class=\"smk-attribute-title\">\n                        Area:\n                    </span>\n                    <span>\n                        <span class=\"smk-value\">\n                            {{feature.properties.Accom_Area}}\n                        </span>\n                    </span>\n                </div>\n            </div>\n            <div class=\"smk-attribute\">\n                <div class=\"smk-attribute\">\n                    <span class=\"smk-attribute-title\">\n                        Start date:\n                    </span>\n                    <span>\n                        <span class=\"smk-value\">\n                            {{feature.properties.MRDT_Start_Date}}\n                        </span>\n                    </span>\n                </div>\n            </div>\n         <!---->\n        </div>\n    </div>\n</div>"
```

Comment:
Continuing ... There's Date.parse(), but it only takes ISO 8601 format (which this isn't). There's always using substrings with separators in between too. But maybe this is getting into the weeds, and the customer hasn't specifically asked for it, and the date is readable as-is. I'd say, if you know of a cheap and reliable way of formatting dates, consider doing so.

Reply:
Thanks for the feedback.
- I agree that formatting the date would be nice, but the customer changed the underlying data type from an epoch date style to this numeric/string representation which as you noticed is not easy to work with. There is no easy way to reliably parse this date format in the context of displaying attributes in this view unfortunately. My plan is to ask the customer to supply us with updated data that uses the epoch style at our meeting with them tomorrow. This would allow us to display nicely formatted dates.
- As far as the /n and space characters, these allow the HTML to be displayed in a friendly, readable format in the SMK Editor app versus an unintelligible string on a single line. 
- Good eye to catch the slight change in spacing in the table. The changes in this PR apply css classes consistently to all row in the table leading to subtle changes (ie the style of the last two rows now matches the style of the first two rows)

Reply:
Thanks for explaining the formatting! I see it now in smk-cli when looking at a layer, going to the Template tab, and choosing "Raw" instead of "Styled". Good to know.